doc/integrator/object_editing.rst

```diff
@@ -45,3 +45,25 @@ Features
         * Cut from: Cut the feature with a geometry from another layer. The
           layer needs WFS and a copyable Metadata.
     * FeatureQuery showing a FeatureWindow
+
+Configuration
+-------------
+
+The layer must be editable.
+
+Copy / cut: The layer from where one wants to copy / cut from has to be:
+
+    * Available as WFS layer
+    * Metadata "copyable" (checked) set on the layer
+
+Add the following configuration int the ``vars.yaml`` file to change the triangle size:
```

Comment:
"int the" -> "in the"

Reply:
the triangle size -> the size of the triangle 
(alternatively: the triangle's size)